src/TraceExtension.php

```
@@ -0,0 +1,48 @@
+use PHPUnit\Event\Test\BeforeTestMethodCalled;
+use PHPUnit\Event\Test\AfterTestMethodCalled;
+use PHPUnit\Event\Test\BeforeTestMethodCalledSubscriber;
+use PHPUnit\Event\Test\AfterTestMethodCalledSubscriber;
+use Koriym\XdebugMcp\TraceHelper;
+
+/**
+ * PHPUnit 10+ Extension for selective tracing
+ */
+class TraceExtension implements Extension
+{
+    public function bootstrap(Configuration $configuration, Facade $facade, ParameterCollection $parameters): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$configuration'. (Unused Code Rules)

(UnusedFormalParameter)